src/algebra/pointwise.lean

```diff
@@ -470,6 +470,10 @@ lemma image2_smul [has_scalar α β] {t : set β} : image2 has_scalar.smul s t =
 lemma mem_smul [has_scalar α β] {t : set β} : x ∈ s • t ↔ ∃ a y, a ∈ s ∧ y ∈ t ∧ a • y = x :=
 iff.rfl
 
+lemma mem_smul_of_mem [has_scalar α β] {t : set β} {a} {b} (ha : a ∈ s) (hb : b ∈ t) :
```

Comment:
Maybe `smul_mem_smul_of_mem`? But it's okay.

Reply:
The version for `has_scalar α (set β)` is called `smul_mem_smul_set`. This version is `has_scalar (set α) (set β)`.

Perhaps this should be
```suggestion
lemma smul_mem_smul_set' [has_scalar α β] {t : set β} {a} {b} (ha : a ∈ s) (hb : b ∈ t) :
```